.travis.yml

```diff
@@ -10,6 +10,13 @@ matrix:
     include:
         - python: "3.7"
           dist: xenial
+        # This tests without [mruwidget] extra
+        - python: "3.7"
```

Comment:
I've recently learned that you can name matrix jobs on Travis:

```yaml
matrix:
  include:
    - name: "test without [mruwidget] extra"
      python: 3.7
      ...
```

It might make the jobs easier to distinguish in the UI.

Reply:
That this a good feature, thanks for the hint.